### ISSUE TITLE
Make it compatible with yarn. Just DO IT

### ISSUE THREAD

**Issue by prenaudin** [antoinelyset/colossus-browser#1]
![screen shot 2016-10-12 at 17 20 44](https://cloud.githubusercontent.com/assets/887094/19316048/4347cd2a-90a0-11e6-8c30-ed2102e23ce1.png)

node version 5.3.0
yarn version 0.15.1

```
client:47./~/colossus-browser/dist/colossus.js
Module not found: Error: Cannot resolve module 'faye/browser/faye-browser' in /Users/prenaudin/Development/Azendoo/az_app_2/node_modules/colossus-browser/dist
resolve module faye/browser/faye-browser in /Users/prenaudin/Development/Azendoo/az_app_2/node_modules/colossus-browser/dist
  looking for modules in /Users/prenaudin/Development/Azendoo/az_app_2/assets/test
    /Users/prenaudin/Development/Azendoo/az_app_2/assets/test/faye doesn't exist (module as directory)
  looking for modules in /Users/prenaudin/Development/Azendoo/az_app_2/assets/src/javascripts
    /Users/prenaudin/Development/Azendoo/az_app_2/assets/src/javascripts/faye doesn't exist (module as directory)
  looking for modules in /Users/prenaudin/Development/Azendoo/az_app_2/assets/src
    /Users/prenaudin/Development/Azendoo/az_app_2/assets/src/faye doesn't exist (module as directory)
  looking for modules in /Users/prenaudin/Development/Azendoo/az_app_2/node_modules
    resolve 'file' or 'directory' browser/faye-browser in /Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye
      resolve file
        /Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser doesn't exist
        /Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser doesn't exist
        /Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser.js doesn't exist
        /Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser.eco doesn't exist
        /Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser.jade doesn't exist
        /Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser.coffee doesn't exist
        /Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser.cjsx doesn't exist
        /Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser.json doesn't exist
        /Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser.scss doesn't exist
      resolve directory
        /Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser doesn't exist (directory default file)
        /Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser/package.json doesn't exist (directory description file)
[/Users/prenaudin/Development/Azendoo/az_app_2/assets/test/faye]
[/Users/prenaudin/Development/Azendoo/az_app_2/assets/src/javascripts/faye]
[/Users/prenaudin/Development/Azendoo/az_app_2/assets/src/faye]
[/Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser]
[/Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser]
[/Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser.js]
[/Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser.eco]
[/Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser.jade]
[/Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser.coffee]
[/Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser.cjsx]
[/Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser.json]
[/Users/prenaudin/Development/Azendoo/az_app_2/node_modules/faye/browser/faye-browser.scss]
 @ ./~/colossus-browser/dist/colossus.js 4:4-50
```

**Comment by prenaudin** [comment]
ping @dfournier 
pong @antoinelyset

**Comment by davidfou** [comment]
Fix with `v0.9.0`
